assets/src/scss/layout/_page-header.scss

```diff
@@ -23,23 +23,27 @@
 }
 
 .page-header {
-  color: $grey-80;
-  padding-top: $n90;
-  margin-bottom: $space-md;
-  position: relative;
+  _-- {
+    color: $grey-80;
+    background: inherit;
```

Comment:
Added it with `inherit` because it's customized by some themes, but by default has the parent's background.